### PR TITLE
Merge changes from finalized release 2.22.0

### PR DESCRIPTION
**Pull request by mokagio**

### CODE REVIEW

**Review by codebykat — APPROVED** [review]
👍

**Comment by codebykat** [comment]
@mokagio Gonna leave it to you to merge this because I can't remember if we should rebase or squash it 😕 I think it should be squashed? Previously we may have used [a single merge commit](https://github.com/Automattic/simplenote-electron/pull/3035).

**Comment by mokagio** [comment]
Closing in favor of https://github.com/Automattic/simplenote-electron/pull/3233

**Pull request closed without merging**
